package.json

```diff
@@ -28,25 +28,25 @@
     "checkPeerDependencies": "node ./scripts/checkPeerDependencies.js"
   },
   "dependencies": {
-    "eslint-config-prettier": "^2.1.1"
+    "eslint-config-prettier": "^2.3.0"
   },
   "devDependencies": {
     "babel-eslint": "^7.2.2",
-    "eslint": "^3.19.0",
-    "eslint-plugin-babel": "^4.1.1",
-    "eslint-plugin-eslint-comments": "^1.0.0",
-    "eslint-plugin-flowtype": "^2.33.0",
-    "eslint-plugin-import": "^2.3.0",
+    "eslint": "^4.5.0",
+    "eslint-plugin-babel": "^4.1.2",
+    "eslint-plugin-eslint-comments": "^1.0.3",
+    "eslint-plugin-flowtype": "^2.35.0",
+    "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jest": "^20.0.3",
-    "eslint-plugin-jsdoc": "^3.1.0",
+    "eslint-plugin-jsdoc": "^3.1.2",
     "eslint-plugin-moment-utc": "^1.0.0",
-    "eslint-plugin-prettier": "^2.1.1",
+    "eslint-plugin-prettier": "^2.2.0",
     "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-react": "^7.0.1",
+    "eslint-plugin-react": "^7.3.0",
     "eslint-plugin-react-native": "^2.3.2",
```

Comment:
Be nice also to upgrade to `eslint-plugin-react-native` 3.0.0
Perhaps can be done as a separate PR though.

Reply:
Agree with this. It looks like the jump from 2.3.2 to 3.0.0 was because of some updates to their own packages including going ESLint 3 -> 4. There's not been any new rules added though, so it's just a case of updating the package.

(Unless there was some problem with it)